policies/main.rego

```diff
@@ -6,12 +6,22 @@ deny = {denial |
 	denial := data.policies[policy].deny[_]
 }
 
+future_deny = {denial |
```

Comment:
I actually don't understand this. It's not a function, it's a comprehension. Got that part. I think this is basically a way to call `skip_not_in_effect` on all rules that have `deny` evaluated to `true`, yes?

I guess we don't really have to save the return value to `future_deny`, but we do so to help mock tests?

Reply:
Oh! It's something we can query for and provide a report. Very nice